### DISCUSSION TITLE
스터디 - 11주차 목록

### DISCUSSION THREAD

**Discussion by vi2920va** [vi2920va/FE_interview#98] [General]
> 📢11주차는 리팩토링 또는 스터디 하실 걸 적어주세요.
🏥 회고록 작성은 없애고, 말로 하는 걸로 변경되었습니다.

## 시간표
2022.01.28(목) - 오후 9시
21:00 ~ 21:15 팀원들 PR 남긴 거 리뷰 시간
21:15 ~ 21:35 모의 면접,1:1, 1:다로 모의면접 진행(상대방이 공부한 것을 기준으로 2개 이상 질문 진행)
21:35 ~ 21:50 모의 면접 진행이 끝난 후, 토론 장소에 모여서 다른 분들에게 궁금했던 걸 토론을 진행
21:50 ~ 22:00 프론트엔드 모의 면접 질문 정리에 작성 후, 돌아가면서 소감 한마디 하고 스터디 종료 

## 동기
- Network - 애플리케이션 레이어

## 두현
- GraphQL - Summary

## 수민
- React - 아토믹 디자인

**Comment by ki7348** [comment]
동기
* Network - 애플리케이션 레이어
